tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -77,6 +77,12 @@ def validate_environment(is_local: bool, use_remote_secrets: bool):
 )
 @click.option("--modified/--not-modified", help="Only test modified connectors in the current branch.", default=False, type=bool)
 @click.option("--concurrency", help="Number of connector tests pipeline to run in parallel.", default=5, type=int)
+@click.option(
+    "--execute-timeout",
+    help="The maximum time in seconds for the execution of a Dagger request before an ExecuteTimeoutError is raised. Passing None results in waiting forever.",
+    default=None,
```

Comment:
Is a 1h default not good for all types of tasks (except maybe the nightly tests?)

Reply:
I wanted to make this specific to publish as until now I haven't been setting timeouts. It should indeed be good for all types of task but to reduce moving parts I prefer to not set the timeout globally.